xray/analyze.py

```diff
@@ -147,7 +148,12 @@ def packet_latency(self, ax):
         ax.hist(buckets, color="blue", bins=num_buckets)
 
     def dropped_packets(self, ax):
-        num_buckets = 100
+        if self.count >= 100:
```

Comment:
If the number of packets (`self.count`) was less than 100, the `bucket_size` would be 0, and the subsequent `/ bucket_size` on line 161 would lead to a division by zero exception. This adjusts the bucket sizes to work for smaller sample sizes